### PR TITLE
gcab 0.6

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/gcab.rb [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vala" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gobject-introspection"
 
   # work around ld not understanding --version-script argument
   # upstream bug: https://bugzilla.gnome.org/show_bug.cgi?id=708257
   patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/gcab.rb]: Doesn't look like this patch is getting accepted upstream. Can you check if it was submitted and, if not, submit it and, if so, see if we can replace it with e.g. an `inreplace` (or see if it's even still needed). Thanks!

Comment (dunn) [inline Library/Formula/gcab.rb]: Followed up on the original bug report, hopefully I'll hear back about accepting the patch.

**Comment by dunn** [comment]
The developers [will merge a patch](https://bugzilla.gnome.org/show_bug.cgi?id=708257#c5) that removes the entire line `-Wl,--version-script=${srcdir}/libgcab.syms`. I'd need to familiarize myself with autotools stuff before being able to create the patch, so if anybody wants to jump in and submit it that'd be great.

**Comment by bfontaine** [comment]
This PR needs to be rebased on top of `master`.

**Comment by dunn** [comment]
Closing since I haven't learned autotools yet and #49959 was merged.

**Pull request closed without merging**